discovery/test/unit_tests/test_detection.py

```diff
@@ -7,6 +7,7 @@
 from discovery import Discovery
 from utils.datagen import FakeDataGen
 
+
```

Comment:
this is why I hate commit all, please don't do this, this is how we end up with merge conflicts

Reply:
Nonono, I added this here on purpose. There must be 2 blank lines between imports and the rest of the file

Reply:
if you want to make structural changes to files outside of your scope, do it in another branch
what if you had 3 other people with 3 different linters auto-formatting? this is how you drive your scrum master insane